### ISSUE TITLE
tosdr.json not getting updated?

### ISSUE THREAD

**Issue by michielbdejong** [duckduckgo/duckduckgo-privacy-extension#482]
Hi!

First of all, thanks again for all your great work on this extension, it's really helping user stay informed.

I have a question - it seems https://github.com/duckduckgo/duckduckgo-privacy-extension/blob/develop/shared/data/tosdr.json was last updated 13 months ago. But as you can see from https://github.com/tosdr/tosdr.org/commits/master/index/services.json 
we do update https://tosdr.org/index/services.json more regularly than that, so I think maybe our changes there are not getting picked up by your build script (anymore). Are you still running https://github.com/duckduckgo/duckduckgo-privacy-extension/blob/master/scripts/tosdr.js as part of your release process?

**Comment by JustinBack** [comment]
Yep it seems to be outdated. However the process still seems to be in the make file, is there any comment on this?

**Comment by JustinBack** [comment]
> 
> 
> Yep it seems to be outdated. However the process still seems to be in the make file, is there any comment on this?

Scratch that, we are commented out

https://github.com/duckduckgo/duckduckgo-privacy-extension/blob/334398560f2acd65d61a83e50ee790c64e1f4e4a/Gruntfile.js#L162-L164
